### PR TITLE
ENH decimal parameter to to_latex and to_html, #12031

### LINKED ISSUE #12031: ENH missing decimal parameter in .to_latex and .to_html
`.to_csv` has a `decimal` parameter to specify the decimal mark for floats since version 0.16

It would be nice to have the same thing in `.to_latex` and `.to_html`.

I'll work on a PR.

This issue is a spin-off of #7061, in which @TomAugspurger  said:

> There’s a *possibility* that we’ll be able to replace some of the `to_latex` code with a Jinja template, similar to the Style stuff. So don’t spend too much time on it :)

### PR DESCRIPTION
**Pull request by nbonnotte**
- [x] closes #12031
- [x] tests added / passed
- [x] passes `git diff upstream/master | flake8 --diff`
- [x] whatsnew entry

I just pass on the `decimal` parameter. All the machinery was already in place.

### CODE REVIEW

File: doc/source/whatsnew/v0.18.1.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Other API Changes
+^^^^^^^^^^^^^^^^^
+
```
Comment (jreback) [inline doc/source/whatsnew/v0.18.1.txt]: this is ok for 0.18.0

File: pandas/core/frame.py [hunk 2]
```diff
@@ -1515,7 +1515,8 @@ def to_html(self, buf=None, columns=None, col_space=None, colSpace=None,
         max_cols : int, optional
             Maximum number of columns to show before truncating. If None, show
             all.
-
+        decimal : string, default '.'
```
Comment (jreback) [inline pandas/core/frame.py]: add a ``versionaddedd`` tag

File: pandas/core/frame.py [hunk 3]
```diff
@@ -1568,6 +1570,8 @@ def to_latex(self, buf=None, columns=None, col_space=None, colSpace=None,
             characters in column names.
         encoding : str, default None
             Default encoding is ascii in Python 2 and utf-8 in Python 3
+        decimal : string, default '.'
+            Character recognized as decimal separator, e.g. ',' in Europe
```
Comment (jreback) [inline pandas/core/frame.py]: same here

File: pandas/tests/test_format.py [hunk 4]
```diff
@@ -760,6 +760,33 @@ def test_to_html_unicode(self):
         expected = u'<table border="1" class="dataframe">\n  <thead>\n    <tr style="text-align: right;">\n      <th></th>\n      <th>A</th>\n    </tr>\n  </thead>\n  <tbody>\n    <tr>\n      <th>0</th>\n      <td>\u03c3</td>\n    </tr>\n  </tbody>\n</table>'
         self.assertEqual(df.to_html(), expected)
 
+    def test_to_html_decimal(self):
+        df = DataFrame({'A': [6.0, 3.1, 2.2]})
```
Comment (jreback) [inline pandas/tests/test_format.py]: add the issue number as a comment

File: pandas/core/frame.py (hunk 3 above)
Comment (nbonnotte) [inline pandas/core/frame.py]: done

**Comment by nbonnotte** [comment]
@jreback All set!

**Comment by jreback** [comment]
thanks @nbonnotte

**Pull request closed without merging**